pkg/apis/batch/types.go

```diff
@@ -39,6 +39,15 @@ type Job struct {
 	Status JobStatus `json:"status,omitempty"`
 }
 
+func (j *Job) IsJobFinished() bool {
```

Comment:
Finally, make sure to update `pkg/controller/job/controller.go` to use the helper instead of the private `isJobFinished`.